backends/qualcomm/serialization/schema.fbs

```diff
@@ -177,7 +181,10 @@ table QnnExecuTorchOptions {
   shared_buffer:bool;
 
   /// Is model from qnn context binary
-  is_from_context_binary: bool;
+  is_from_context_binary:bool;
```

Comment:
oh is it only for the custom op solution?

Reply:
Yes, we need this flag to guarantee the order of graph IOs.